main.cpp

```diff
@@ -12,6 +13,8 @@
 using namespace viam::sdk;
 
 int main(int argc, char** argv) {
+    const Instance instance;
```

Comment:
Oops. I missed this in #35. I thought I'd tested adequately but I guess I failed to rebuild the docker image correctly.